connectors/src/connectors/confluence/temporal/client.ts

```diff
@@ -18,7 +18,8 @@ import { getTemporalClient } from "@connectors/lib/temporal";
 
 export async function launchConfluenceFullSyncWorkflow(
```

Comment:
Maybe call it SyncWorkflow vs FullSyncWorkflow which kinda hints that it's one time?